source/custom/nixnet_service.custom.cpp

```diff
@@ -1056,7 +1056,7 @@ ::grpc::Status NiXnetService::DbGetDatabaseList(::grpc::ServerContext* context,
     response->set_status(status);
     if (status_ok(status)) {
       response->set_alias_buffer(alias_buffer.c_str());
-      response->set_file_path_buffer(file_path_buffer.c_str());
+      response->set_filepath_buffer(file_path_buffer.c_str());
```

Comment:
This is custom_code for the `DbGetDatabaseListSizes` thing, right? it would be nice to add comments on why these are customcode (just noticing now that I forgot why). A lot of them are pretty obviously custom, but some of these look very similar to generated code.

Reply:
Updated documentation for all of the custom implementations.